templates/business/email.txt

```diff
@@ -0,0 +1,10 @@
+Downtime for period {{ start }} - {{ end }}
+
+{{ 'Subject'|stringformat:"-30s" }} {{ 'Incidents'|stringformat:"-10s" }} {{ 'Downtime'|stringformat:"-20s" }} {{ 'Availability' }}
+------------------------------------------------------------------------------------------{% for record in records %}
+{{ record.subject|stringformat:"-30s" }} {{ record.incidents|length|stringformat:"-10s" }} {{ record.downtime|stringformat:"-20s" }} {{ record.availability }}{% empty %}
```

Comment:
Isn't there a  way to get the longest string from the list to feed into this formatter? My test e-mail had a skewed first line because the first device had a 31 character DNS name.